ingestion-beam/src/main/java/com/mozilla/telemetry/heka/File.java

```diff
@@ -0,0 +1,4 @@
+package com.mozilla.telemetry.heka;
```

Comment:
I assume including this file in the PR is accidental?

Reply:
argh, how did that get in there :) will remove